### PR TITLE
Adds Xcake integration.

### PR DESCRIPTION
**Pull request by jcampbell05**
Adds ability to invoke the `xcake` command for Xcake https://github.com/jcampbell05/xcake/

### CODE REVIEW

**Comment by KrauseFx** [comment]
Wups, somehow missed that notifications, reviewing the PR now :+1:

File: lib/fastlane/actions/xcake.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module Fastlane
+  module Actions
+    class XcakeAction < Action
+      def self.run(params)
+        Actions.verify_gem!('xcake')
+        Actions.sh('xcake')
```
Comment (KrauseFx) [inline lib/fastlane/actions/xcake.rb]: While this works, you could potentially change this to

```ruby
require 'xcake'
Xcake::Command.run
```
or similar. 

What do you think? By using the Ruby code, the user will get the actual stack trace if something goes wrong.

**Comment by KrauseFx** [comment]
Hey, this looks great, thanks :+1: I added one comment before it's ready to be merged. Could you also squash the commits before I merge it in?

File: lib/fastlane/actions/xcake.rb (hunk 1 above)
Comment (jcampbell05) [inline lib/fastlane/actions/xcake.rb]: Makes sense :) I'll alter this.

**Comment by jcampbell05** [comment]
@KrauseFx Oops seem's I don't know how to squash ! I'll fix this.

**Pull request closed without merging**